OrganizationSearch/FindOrganization.js

```diff
@@ -40,6 +48,12 @@ export const FindOrganization = ({ selectVendor, ...rest }) => {
 
   const { fetchOrganizations } = useOrganizations();
 
+  const searchableIndexes = isDonorsEnabled ? donorsSearchableIndexes : organizationSearchableIndexes;
```

Comment:
all of these should come from props - with default values

Reply:
Got it.